### DISCUSSION TITLE
Is there any way to pass flags in mitmdump scripts?

### DISCUSSION THREAD

**Discussion by rohitcoder** [mitmproxy/mitmproxy#6072] [Q&A]
Hi, 

I'm trying to pass a flag let's say ``--myflag`` which should be accessible inside ``save.py``, i was trying to below-mentioned command, but it's not working for me. I want to do some dynamic stuff inside save.py using flags.

``
mitmdump --listen-port 8081 -s save.py --myflag
``

**Comment by Prinzhorn** [comment]
https://docs.mitmproxy.org/stable/addons-options/
